style/index.css

```diff
@@ -88,7 +93,7 @@ label {
 }
 
 .signup-form {
-    width: 333px;
```

Comment:
din cauza ca noi am adaugat aici la `form` element `width` cel mai mare element dar nu am luat in considerare `border` si `margin` care vin by default de la elementul `input`